src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -320,7 +321,8 @@ public function getAttributes()
     {
         return [
             'data-editor' => 'tinyMCE', // Register ss.editorWrappers.tinyMCE
-            'data-config' => Convert::array2json($this->getConfig())
+            'data-config' => Convert::array2json($this->getConfig()),
+            'style' => 'height:350px'
```

Comment:
Ah cool, have you tested with just `style`? it seems to work just as well with and without the `min_height`, I think
The main reason I'm looking at removing `min_height` is because I can resize the box to be smaller than 350px if I wanted to.